portal/src/main/webapp/css/oql-editor.css

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2016 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <h2ttp://www.gnu.org/licenses/>.
+ */
+ 
+
+/* #gene_list input div to query gene symbols */
+#gene_list {
+  background-color: #eeeeee;
+  width: 75%;
+  font-weight: bold;
+  font-size: 15px;
+  text-transform: uppercase;
+  font-family: verdana, arial, sans-serif;
+  height: 115px;
+  -webkit-box-shadow: inset 2px 2px 0px 0px rgba(134,133,132,1);
+  -moz-box-shadow: inset 2px 2px 0px 0px rgba(134,133,132,1);
+  box-shadow: inset 2px 2px 0px 0px rgba(134,133,132,1);
+  overflow: auto;
+  padding: 4px;
+  
+  white-space: pre-wrap;       /* css-3 */
+  white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+  white-space: -pre-wrap;      /* Opera 4-6 */
+  white-space: -o-pre-wrap;    /* Opera 7 */
+  word-wrap: break-word;       /* Internet Explorer 5.5+ */
+}
+
+#gene_list:empty:not(:focus):before{
+	color: #898989;
+	content:attr(placeholder);
+}
+
+
+/* OQL Menu */
+
+.hidden-menu {
```

Comment:
All of these class names seem to me too general, I think they should all be prefixed by `oql-editor` e.g. `.oql-editor-hidden-menu`